hpms/src/main/java/gov/cms/ab2d/hpms/hmsapi/HPMSOrganizationInfo.java

```diff
@@ -0,0 +1,18 @@
+package gov.cms.ab2d.hpms.hmsapi;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
```

Comment:
Do you need all the constructors used here? Same with the other classes

Reply:
With adding the AllArgsAnnotation, I was concerned that marshalling would break, hence I added the NoArgs too.  

I just did some testing and confirmed that NoArgs is not necessary and have removed it.